build-aux/summit.cmake

```diff
@@ -23,14 +23,11 @@ set(MPIEXEC_PREFLAGS "-a 1 -g 1 -c 5" CACHE STRING
 set(SMPIARGS_FLAG_NOMPI "--smpiargs=none" CACHE STRING
   "Spectrum MPI argument list flag for serial tests.")
 # Let's keep this option in case we need it again in the future.
-set(SMPIARGS_FLAG_MPI "" CACHE STRING "Spectrum MPI argument list flag for MPI tests.")
-
-# When we want to us a cuda visible devices restriction we need this flag
-set(SMPIARGS_FLAG_MPI_CVD "--smpiargs=-gpu" CACHE STRING 
```

Comment:
Change the name to _MGPU or something and don't remove it.  I have to build and test the code on more systems than a laptop and summit and its is useful to be able to partition the multigpu tests.  This is a useful distinction as long as our tests target 1 node.

Reply:
the additional flags can be easily added to the SMPIARGS_FLAG_MPI, as they don't need to differ between tests, even if the mpi implementation is not spectrum (sloppy naming on our side).